### ISSUE TITLE
[BUG] 2.0: JavascriptException when updating chart via on_range_change with legend turned on + line created

### ISSUE THREAD

**Issue by amirzamli** [louisnw01/lightweight-charts-python#390]
### Expected Behavior

Just updating the chart, no crash

### Current Behaviour

Exception in thread Thread-2 (loop):
Traceback (most recent call last):
  File "c:\documents\1_projects\lightweight-charts-python\lightweight_charts\chart.py", line 89, in loop
    window.evaluate_js(arg)
  File "\.venv-win\lib\site-packages\webview\window.py", line 50, in wrapper
    return function(self, *args, **kwargs)
  File "\.venv-win\lib\site-packages\webview\window.py", line 455, in evaluate_js
    raise JavascriptException(result)
webview.errors.JavascriptException: {'name': 'TypeError', 'stack': "TypeError: Cannot read properties of undefined (reading 'value')\n    at http://127.0.0.1:61435/bundle.js:1:5434\n    at Array.forEach (<anonymous>)\n    at o.legendHandler (http://127.0.0.1:61435/bundle.js:1:5126)\n    at http://127.0.0.1:61435/lightweight-charts.js:7:6631\n    at Array.forEach (<anonymous>)\n    at k.m (http://127.0.0.1:61435/lightweight-charts.js:7:6617)\n    at Object.o (http://127.0.0.1:61435/lightweight-charts.js:7:157503)\n    at http://127.0.0.1:61435/lightweight-charts.js:7:6631\n    at Array.forEach (<anonymous>)\n    at k.m (http://127.0.0.1:61435/lightweight-charts.js:7:6617)", 'message': "Cannot read properties of undefined (reading 'value')"}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Amir Z\AppData\Local\Programs\Python\Python310\lib\threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "C:\Users\Amir Z\AppData\Local\Programs\Python\Python310\lib\threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "c:\documents\1_projects\lightweight-charts-python\lightweight_charts\chart.py", line 95, in loop
    raise JavascriptException(f"\n\nscript -> '{arg}',\nerror -> {msg['name']}[{msg['line']}:{msg['column']}]\n{msg['message']}")
KeyError: 'line'

### Reproducible Example

```python
import pandas as pd
from lightweight_charts import Chart

chart = Chart()

line = chart.create_line("test")
chart.legend(True)

df1 = pd.read_csv(f"./data/data2.csv")
chart.set(df1)

def on_range_change(chart, bef, aft):
    chart.set(df1)

chart.events.range_change += on_range_change

if __name__ == "__main__":
    chart.show(block=True)
```

### Environment

```markdown
- OS: Windows 10
- Library: lightweight-charts 2.0, Python 3.10
```

**Comment by amirzamli** [comment]
Setting chart.legend(False) resolves the issue

**Comment by louisnw01** [comment]
Fixed in 2.0.1 @amirzamli
